http/src/utils.rs

```diff
@@ -0,0 +1,70 @@
+use hyper::{self, header};
```

Comment:
I think we miss pre-ambles in the entire codebase (probably that was the case for jsonrpc too)